tests/e2e/homework/tests/loginPage.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {test, expect} from '@playwright/test';
+import { PageManager } from '../pageObject/poManager';
+import { variables } from '../pageObject/config';
+
+let pageManager:PageManager;
+
+test.beforeEach('Go to page', async({page}) => {
+    pageManager = new PageManager(page);
+    await pageManager.login.goToPage();
+});
+    test('Login with valid details', async ({page}) => {
+        await pageManager.login.loginUser(variables.validUsername, variables.validPassword);
+        await expect(pageManager.successAlert).toBeVisible();
```

Comment:
If you're checking just for visibility of the element - it's ok. 
(apart from callling the function from pagemanager)

although if we want to check the alert with it's message:

basePage.ts: add generic function for verification
```
    async verifySuccessAlert(expectedText: string) {
        await expect(this.successAlert).toBeVisible();
        await expect(this.successAlert).toHaveText(expectedText);
    }
```

and then

either call it in test with a string of expected text.

or

create another function in start page
```
    async verifySuccessSignInAlert() {
        await this.verifySuccessAlert("Signed in!");
    }
```

Reply:
Choose one method and apply thorugout the solution ;)